commands/orb.js

```diff
@@ -1,6 +1,8 @@
 const fetch = require("node-fetch");
 const Discord = require("discord.js");
 const { asyncHandler } = require("./_helper");
+const { multipleResultsFoundMessage } = require("./_helper")
+const { noResultFoundMessage } = require("./_helper")
```

Comment:
Minor thing. You can add a comma after multipleResultsFoundMessage and simply have noResultFoundMessage to keep this at 1 line of imports per file. Not too concerned about this

Reply:
ok